features/setting_constraints/matching_arguments.feature

```diff
@@ -46,6 +46,31 @@ Feature: Matching arguments
       |     expected: (1, anything, /bar/)               |
       |          got: (1, nil, "other")                  |
 
+  Scenario: Using a RSpec matcher
+    Given a file named "rspec_matcher_spec.rb" with:
+      """ruby
+      RSpec.describe "Using a RSpec matcher" do
+        let(:dbl) { double }
+        before { expect(dbl).to receive(:foo).with(contain_exactly(1, 2)) }
+
+        it "passes when the args match" do
+          dbl.foo([1, 2])
```

Comment:
Given that `contain_exactly` allows the array to be in any order, might be worth doing `db.foo([2, 1])` here -- otherwise, the expectation above doesn't really need the additional flexibility `contain_exactly` provides.